recipes/actflow/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "actflow" %}
+{% set version = "0.3.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/actflow-{{ version }}.tar.gz
+  sha256: 4befb231c4b4addb153c566f24f4da660caecd77e4d4bfaee05bb492f8944e98
+
+build:
+  noarch: python
+  script: 
+    - {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python <3.12
+    - pip
+  run:
+    - python <3.12
+    - gglasso
+    - seaborn
+    - matplotlib-base
+    - numpy <1.24
+    - h5py
+    - statsmodels
+    - nibabel
+    - wheel
+    - ipython
+    - pillow
+    - wheel
+    - wbplot
```

Comment:
The package metadata has:

```
    - python >=3.7
    - gglasso
    - wbplot
    - seaborn
    - matplotlib-base
    - numpy <1.24
    - h5py
    - statsmodels
```

Can you check that?

Reply:
I have removed redundant packages